src/backend/web/templates/account_login_required.html

```diff
@@ -5,8 +5,13 @@
 {% block meta_description %}Login Required{% endblock %}
 
 {% block inline_javascript %}
+<script src="//gstatic.com/firebasejs/8.10.0/firebase-auth.js"></script>
 <script src="/py3_javascript/tba_combined_js.firebase.min.js"></script>
-<script src="//gstatic.com/firebasejs/8.2.1/firebase-auth.js"></script>
+<script>
+  if ({{ auth_use_emulator|tojson }}) {
+    auth.useEmulator("http://localhost:9099");
```

Comment:
- [ ] I'm pretty sure we can't do this in `tba_firebase.js` - but we should check.